modules/unsupported/geojsonstore/pom.xml

```diff
@@ -36,6 +32,18 @@
 			<version>${project.version}</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>com.fasterxml.jackson.core</groupId>
+			<artifactId>jackson-databind</artifactId>
+			<version>2.9.7</version>
+		</dependency>
+		<!-- https://mvnrepository.com/artifact/org.locationtech.jts.io/jts-io-common -->
+        <dependency>
+	       <groupId>org.locationtech.jts.io</groupId>
```

Comment:
Interesting, what is this used for?

Reply:
it's for `org.locationtech.jts.io.geojson.GeoJsonReader` which I'm using to process the geometries.

Reply:
So json-simple goes out from the main entrance and then gets back in from the backdoor :-D
https://github.com/locationtech/jts/blob/master/modules/io/common/pom.xml#L38

Reply:
Another possible alternative is this: https://github.com/bedatadriven/jackson-datatype-jts
Pro, it does not bring in back json-simple and is designed for jackson, cons, seems to have gone un-maintained too and uses and older version of JTS. Again, just thinking out loud.